hw/ip/clkmgr/doc/theory_of_operation.md

```diff
@@ -160,6 +160,12 @@ The following diagram enhances the block diagram to illustrate the overall reset
 
 * [Interface Tables](../../../top_earlgrey/ip/clkmgr/data/autogen/clkmgr.hjson#interfaces)
 
+### Clock Gated Indications for Alert Handler
```

Comment:
Move this above the Hardware Interfaces section?

Reply:
done